sermant-plugins/sermant-flowcontrol/flowcontrol-service/src/main/java/io/sermant/flowcontrol/res4j/chain/handler/XdsBusinessRequestHandler.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2024-2024 Sermant Authors. All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package io.sermant.flowcontrol.res4j.chain.handler;
+
+import io.sermant.flowcontrol.common.core.match.XdsMatchManager;
+import io.sermant.flowcontrol.common.entity.BusinessEntity;
+import io.sermant.flowcontrol.common.util.XdsThreadLocalUtil;
+import io.sermant.flowcontrol.res4j.chain.AbstractChainHandler;
+import io.sermant.flowcontrol.res4j.chain.HandlerConstants;
+import io.sermant.flowcontrol.res4j.chain.context.RequestContext;
+
+/**
+ * The service matches the handler with the highest priority（in addition to surveillance）
```

Comment:
Describe the function of the handler in detail.

Reply:
done